tokens/lib/tokens.json

```diff
@@ -4930,6 +5262,292 @@
           "value": "{typography.body.body14}",
           "type": "typography"
         }
+      },
+      "headings": {
+        "h1": {
+          "value": {
+            "fontFamily": "{fontFamilies.inter}",
+            "fontWeight": "{fontWeights.bold}",
+            "lineHeight": "{lineHeights.7}",
+            "fontSize": "{fontSize.2xl}",
+            "letterSpacing": "{letterSpacing.default}",
+            "textCase": "{textCase.default}",
+            "textDecoration": "{textDecoration.default}"
+          },
+          "type": "typography"
+        },
+        "h2": {
+          "value": {
+            "fontFamily": "{fontFamilies.inter}",
+            "fontWeight": "{fontWeights.bold}",
+            "lineHeight": "{lineHeights.6}",
```

Comment:
Hey did the headings tokens need to be moved down to component level, I think they worked as global tokens? It is causing some error when trying to transpile the tokens: 
![Uploading Screenshot 2023-09-20 at 15.56.20.png…]()

Reply:
They should be used within components, so yeah I believe. I could just duplicate them probably (so they will on both levels)

Reply:
Reason for failing is that in this case the components typography tokens and the global typography tokens have the same name. The names should be unique, say adding st. before the names should fix it